python/openvino_tokenizers/build_tokenizer.py

```diff
@@ -55,7 +55,7 @@ def build_rwkv_tokenizer(
         _get_factory()
         .create(
             "VocabDecoder",
-            [*detokenizer_input.outputs(), *BasePipelineStep.create_string_constant_node(trie_node.vocab)],
+            [*detokenizer_input.outputs(), *create_string_constant_node(trie_node.vocab)],
```

Comment:
`BasePipelineStep` import should be deleted. Please check similar issues with ruff.

Reply:
done